.github/workflows/linux.yml

```diff
@@ -222,6 +222,8 @@ jobs:
 
       - name: Build Debian packages
         run: |
+          /usr/bin/python3 -m pip install -U pip
```

Comment:
BTW, on newer Linux versions it's not allowed to install packages to global env (for me, it complains that system can be broken)
Maybe venv need to be created within docker and exposed instead of `/usr/bin/python`?

Reply:
Created a dedicated 3.10 venv because the Debian packages job seems to be working only with the system Python. Now the dependencies is not installed in the global env.